### DISCUSSION TITLE
ما هي الشروط و المعايير التي نعتمدها في تقييم المصطلحات والمفاضلة بينها؟

### DISCUSSION THREAD

**Discussion by Abdelaziz18003** [vuejs-translations/docs-ar#70] [سؤال]
أرى أنه يتوجب علينا ضبط المعايير التي سوف نعتمدها في اختيار المصطلحات والمفاضلة بينها قبل البدء في اعتماد المصطلحات بشكل رسمي، حتى يكون هناك نظام واضح وشروط محددة في تقييم المصطلح ولا يكون الإختيار حسب الذوق الشخصي الذي سوف يختلف من شخص لآخر.
مرحبا باقتراحاتكم 🌱

**Comment by boussadjra** [comment]
عند اقتراح ترجمة معينة، يجب أن يؤتى بالدلائل و الأمثلة على صوابها.

**Comment by zeyadmoamen1210** [comment]
عند ترجمة فقرة معينة يمكن عدم اللإلتزام بالترجمة الحرفية و يمكن إعادة صياغتها بطريقة سلسة أو إضافة عبارات أخري فيها لا يتعارض مع مضمون الفقرة

**Comment by Abdelaziz18003** [comment]
مقترحاتي هي:

- **الصحة**: أي أن يكون المصطلح صحيحا لغويا
- **الدقة**: أن يكون معبرا عن المعنى في السياق الأصلي ولا يشترط أن يكون ترجمة حرفية، بل تقدم الترجمة المعبرة عن السياق على الترجمة الحرفية إذا لم تكن الترجمة الحرفية معبرة عن المعنى جيدا
- **الشيوع**: أي إذا تساوت المصطلحات في الصحة والدقة يقدم المصطلح الأكثر شيوعا في المشاريع الأخرى المشابهة
- **الإختصار** أي إذا أمكن إيجاد مصطلح مكون من كلمة وآخر مكون من كلمتين وتساويا في كل شيء آخر فإن الأقصر يقدم على الأطول
- **سهولة النطق**: أي إذا تساوى مصطلحان في جميع الشروط الأخرى فإن الأسهل نطقا يقدم على الأصعب

أود لو يشارك الجميع بتعليقاتهم حول هذه المقترحات، زيادةً وتعديلاً أو حتى رفضاً إذا رأيتم أني أخطأت في بعضها

**Reply by boussadjra** [reply]
من أجل أن نسرع الترجمة أو أن لا نعلق مع مصطلح معين، يمكنك وضع المفردة التي تراها مناسبة بين قوسين في انتظار الاتفاق عليها.
